go/consensus/cometbft/light/service.go

```diff
@@ -213,6 +214,7 @@ func (c *client) worker() {
 		providers[0],  // Primary provider.
 		providers[1:], // Witnesses.
 		c.store,
+		cmtlight.PruningSize(config.GlobalConfig.Consensus.Prune.NumLightBlocksKept),
```

Comment:
Would it make more sense to use 0 (disabled pruning) if `config.GlobalConfig.Consensus.Prune.Strategy` is "none"? 
And then also mention in the docs that it defaults to 1000 if pruning is enabled.

Reply:
Good idea, will change to honor the strategy as well.

Reply:
Done.